cypress/integration/publishingFlows/articleBrowserPersistence.js

```diff
@@ -0,0 +1,175 @@
+describe('Article Editor', () => {
+  describe('v1 Editor', () => {
+    beforeEach(() => {
+      cy.testSetup();
+      cy.fixture('users/articleEditorV1User.json').as('user');
+
+      cy.get('@user').then((user) => {
+        cy.loginUser(user).then(() => {
+          cy.visit('/new');
+        });
+      });
+    });
+
+    describe(`revert changes`, () => {
```

Comment:
I'm going to add some comments throughout the tests to make it clearer what's going on.